packages/cozy-harvest-lib/src/components/cards/RunningAlert.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+
+import { isFlagshipApp } from 'cozy-device-helper'
+import Alert from 'cozy-ui/transpiled/react/Alert'
+import AlertTitle from 'cozy-ui/transpiled/react/AlertTitle'
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import ArrowUp from 'cozy-ui/transpiled/react/Icons/ArrowUp'
+
+export const RunningAlert = (): JSX.Element | null => {
+  const { t } = useI18n()
+  const [isVisible, setIsVisible] = useState(isFlagshipApp())
+  const handleClose = (): void => setIsVisible(false)
+
+  return isVisible ? (
+    <Alert
+      className="u-mt-1"
+      block
+      icon={<Icon icon={<ArrowUp />} color="var(--primaryColor)" />}
```

Comment:
@JF-Cozy Is it the API? Can't we use "primary" directly?

Reply:
I'm gonna check anyway

Reply:
@Crash-- updated, the icon inherits the correct colour from the alert component default style currentColor. So I don't need to specify anything here

Reply:
seems to be solved :)